### ISSUE TITLE
Support Deno 1.4.1 & STD 0.70.0

### ISSUE THREAD

**Issue by khrome83** [SachaCR/denock#7]
I started a PR to handle this upgrade, but I got in over my head with some stuff I didn't understand.

In theory, this fork should work with 1.4.0, but for some reason fails. 

https://github.com/use-seedling/denock

@SachaCR can you pull this down and see if you can identify what is wrong with the 5 tests that fail all of a sudden?

I will open this in the meantime, knowing the test fails.

**Comment by SachaCR** [comment]
Yes I saw the other day that test are failing after 1.4.0 update. I'll try to figure out why.

**Comment by andre4ik3** [comment]
Almost two years later, it works on Deno `1.20.4` 🙃 Any hope of the project being improved to be more like the real Nock (like with method chaining) or is it completely abandoned? I wanted to use it to port some existing NodeJS Jest+Nock tests to Deno, but alas, I can't find any Nock alternative for Deno other than this library...

**Comment by SachaCR** [comment]
Hello, sorry I don't have the time to work on this lib anymore. Feel free to fork it if you need to make it evolve.
